x-pack/plugin/inference/qa/rest/build.gradle

```diff
@@ -0,0 +1,15 @@
+apply plugin: 'elasticsearch.internal-yaml-rest-test'
```

Comment:
I don't think we need to create _another_ QA project here. We can just apply this plugin to the the `:x-pack:plugin:inference` project and run these tests there.

Reply:
I thought that was a common pattern - moved directly under the plugin root dir 👍

Reply:
noted, thank you Mark!